core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -639,9 +688,9 @@ object TopicCommand extends Logging {
     private val topicsWithOverridesOpt = parser.accepts("topics-with-overrides",
       "if set when describing topics, only show topics that have overridden configs")
     private val ifExistsOpt = parser.accepts("if-exists",
-      "if set when altering or deleting or describing topics, the action will only execute if the topic exists. Not supported with the --bootstrap-server option.")
+      "if set when altering or deleting or describing topics, the action will only execute if the topic exists.")
```

Comment:
We need to update `checkArgs` to accept them with `bootstrap-server`. As the moment, the tool refuses them with the following error:
```
Option "[if-not-exists]" can't be used with option "[bootstrap-server]"
```